index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+

```

Comment:
@Guha-Pritam , I still do not see ur changes.

- Please create a branch.
- Switch to the new branch
-  Make your code changes
- Add, Commit, Push
- Create a Pull request
- Add me as a reviewer.